src/components/journal/stamps/stamp-selector.tsx

```
@@ -0,0 +1,52 @@
+            defaultValue="miku"
+            onValueChange={(selectedStamp) => setStamp(selectedStamp)}
+            className="w-full grid grid-cols-2 justify-items-center"
+          >
+            {['waves', 'hana', 'neko', 'miku'].map((icon) => (
+              <ToggleGroupItem
+                value={icon}
+                onClick={() => {
+                  setStamp(icon)
+                }}
+                className="border-r even:border-r-0 border-b border-border p-2 rounded-none h-full w-full"
+              >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)